examples/ALKS_BuzzerSongs/pitches.h

```diff
@@ -0,0 +1,90 @@
+

```

Comment:
This seems like it is not your work, you signed a CLA, where you stated everything is written by you, or you gave appropriate credit (where license allows).